Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/README.md

```diff
@@ -2235,3 +2235,117 @@ Gets the status of the download file request.
 |---|---|---|---|---|
 | 3 | HOST_NAME | Completed | C:\Program Files (x86)\Tanium\Tanium Client\Logs\log1.txt | 2020-01-02 15:40:18.052 |
 
+
+### tanium-tr-intel-doc-create
+***
+Add a new intel document to the system by providing its document contents.
+
+
+##### Base Command
+
+`tanium-tr-intel-doc-create`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| intel-doc | Intel Document in OpenIOC format containing hashes | Required |
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Tanium.IntelDoc.AlertCount | Number | The number of alerts that currently exist for this intel. |
+| Tanium.IntelDoc.CreatedAt | Date | The date at which this intel was first added to the system. |
+| Tanium.IntelDoc.Description | String | The description of the intel, as declared in the document or as updated by a user. |
+| Tanium.IntelDoc.ID | Number | The unique identifier for this intel in this instance of the system. |
+| Tanium.IntelDoc.LabelIds | Number | The IDs of all labels applied to this intel. |
+| Tanium.IntelDoc.Name | String | The name of the intel, as declared in the document or as updated by a user. |
+| Tanium.IntelDoc.UnresolvedAlertCount | Number | The number of unresolved alerts that currently exist for this intel. |
+| Tanium.IntelDoc.UpdatedAt | Date | The date when this intel was last updated. |
+
+
+##### Command Example
+```!tanium-tr-intel-doc-create intel-doc=${IntelDoc}```
```

Comment:
Can `${IntelDoc}` be changed to a number (as it is relevant only to your script)? I think here 2 is the relevant option.

Reply:
The `intel-doc` parameter here is a string in ioc format. I see in TTRv2 the command `tanium-tr-intel-doc-create` has an argument `file_extension` which supports yara and stix also. I could not upload the IntelDoc.xml file here because it's not supported. Here are the contents:

`<?xml version="1.0" encoding="us-ascii"?>
<ioc xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:xsd="http://www.w3.org/2001/XMLSchema" id="1" last-modified="2021-10-19T19:50:59"
    xmlns="http://schemas.mandiant.com/2010/ioc">
    <short_description>short description</short_description>
    <description>description</description>
    <authored_by>author</authored_by>
    <authored_date>2021-10-19T19:50:59</authored_date>
    <links />
    <definition>
        <Indicator operator="OR" id="872d965b-3f48-46ff-b5d2-3049d327fa05">
            <IndicatorItem id="cb1e250b-1938-4671-a5c0-717d29708ad9" condition="is">
                <Context document="FileItem" search="FileItem/Md5sum" type="mir" />
                <Content type="md5">17B159A73E4FA721AF8501E8E9809C73</Content>
            </IndicatorItem>
        </Indicator>
    </definition>
</ioc>`